modules/serverless-go-template.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-go-functions.adoc
+
+[id="serverless-go-template_{context}"]
+= Golang function template structure
+
+When you create a Golang function using the `kn func` CLI, the project directory looks like a typical Go project, with the exception of an additional `func.yaml` configuration file.
```

Comment:
Should this be "using the `kn func` command" ?

Reply:
Changed to `kn` CLI